docs/book/reference/faq.md

```diff
@@ -6,64 +6,42 @@ description: Find answers to the most frequently asked questions about ZenML.
+Take a look at our [examples](https://github.com/zenml-io/zenml/tree/main/examples) directory, which showcases detailed examples for each integration that ZenML supports out-of-the-box.
 
-The ZenML team and community are constantly working to include more tools and integrations to the above list (check out
-the [roadmap](https://zenml.io/roadmap) for more details). You can [upvote features](https://zenml.io/discussion) you'd
-like and add your ideas to the roadmap.
+The ZenML team and community are constantly working to include more tools and integrations to the above list (check out the [roadmap](https://zenml.io/roadmap) for more details). You can [upvote features](https://zenml.io/discussion) you'd like and add your ideas to the roadmap.
 
-Most importantly, ZenML is extensible, and we encourage you to use it with whatever other tools you require as part of
-your ML process and system(s). Check out [our documentation on how to get started](../introduction.md)
-with extending ZenML to learn more!
+Most importantly, ZenML is extensible, and we encourage you to use it with whatever other tools you require as part of your ML process and system(s). Check out [our documentation on how to get started](../introduction.md) with extending ZenML to learn more!
 
 #### How can I make ZenML work with my custom tool? How can I extend or build on ZenML?
 
-This depends on the tool and its respective MLOps category. We have a full guide on this
-over [here](/docs/book/platform-guide/set-up-your-mlops-platform/implement-a-custom-stack-component.md)!
+This depends on the tool and its respective MLOps category. We have a full guide on this over [here](../user-guide/advanced-guide/custom-stack-solutions/implement-a-custom-stack-component.md)!
 
 #### How can I contribute?
 
-We would love to develop ZenML together with our community! The best way to get started is to select any issue from
-the [`good-first-issue` label](https://github.com/zenml-io/zenml/labels/good%20first%20issue). If you would like to
-contribute, please review our [Contributing Guide](https://github.com/zenml-io/zenml/blob/main/CONTRIBUTING.md) for all
-relevant details.
+We would love to develop ZenML together with our community! The best way to get started is to select any issue from the [`good-first-issue` label](https://github.com/zenml-io/zenml/labels/good%20first%20issue). If you would like to contribute, please review our [Contributing Guide](https://github.com/zenml-io/zenml/blob/main/CONTRIBUTING.md) for all relevant details.
 
 #### How can I learn more about MLOps?
 
-Check out our [ZenBytes](https://github.com/zenml-io/zenbytes) repository and course, where you learn MLOps concepts in
-a practical manner with the ZenML framework. Other great resources are:
+Check out our [ZenBytes](https://github.com/zenml-io/zenbytes) repository and course, where you learn MLOps concepts in a practical manner with the ZenML framework. Other great resources are:
```

Comment:
Hasn't this somehow been deprecated? Not even sure whether it works still?

Reply:
I removed the question altogether.